multinode_installation/vagrant_libvirt/.vagrant/machines/datacenter/libvirt/private_key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEowIBAAKCAQEAyiAIlajV9P6X/fTdIu0dDMC2zD3T1L8e+WJCUUy37pDzdkGd
+k/vI4G3w7AIye0lMXbatYXvM3EVJ2khEex38K1mt8iP7HiG3IF80otOHb6NvWLe+
+laRK1+cNT1y88z8CD2QSjvGCqIUaoIR3D73CWpyTjKEI/5HKcUQwVN7TsfOGbQsS
+ApLq4ebn0xy1oTyLa6AKRhGmFayHem1H+GjPubPD5KBxi1yzVbT2qkWR+FMpv+ky
+M5YfStNcpeVmGkuE3Hm6PO69JCKGe8vBN8iKli3AGonUlJDO8kCFlbC8dXD1GDcq
+cZXSS4/jh/r9sVMqNf1KF8KWTAsCMXn0UqwgjwIDAQABAoIBACK8lJ0b0gFzy61C
+asq401hAummpaoFSg0E6qMkOFUIzuROmvS0UhXZEjrYQvpo/g5dLcMqjGxbz9XhF
+GTfLdG4C0Ikfs+V+x9kRY5tkHQ+Ll/Q4R9Gx1Yb7V2c/Od3YA0Of4CyexuStU2Eo
+wsl6xC8MX7M4bbJ2XMNQg7niPaGEOVRCkZO/ad/AgmLho+/cmQWh1T/Hodu0XFF9
+1WHB1rWmgVxwJq1gbec1daefqKLEjjxItbP9Im1SamffayMg5R309B2QkjvnWJsm
+aRNdr2UU/xXiZaH8cy4WmshCY+9jp+rAU8fq/YQq2dY6M+Uy2WvOEBwpc6cEmmsd
+JkjhSbkCgYEA8HaISE4es3h2sdb9G1kovl3x5eS4OJsuwVZyTg0PmJFdHxbpj6eK
+4PfuFaZU4ju78BMzaA1PpL8tC72FsEto0+uXyolBbcO9QO9VuUzWb7csuvqyh1M3
+pd7QE55LraPsR2Vj4uf027jFqpVp+EWDzqWyGYg+EoflP4IFSn9Ko2UCgYEA1y9b
+8n8tPdFZEb7YPKAeU8mWKlgZ6FKYr1ivp31PnPNxSEBNx46yLRwgXgZMAcHKl+n3
+gBTkHp4teHkMv4BJcdkJtgDN+gRrcUS/8QL/rgacvA1rWT9kRXPcnee6+F3tKHkQ
+i6bXUl+YYIR3BB3PEiIXUskrSKL6ys5gm2onZuMCgYBLk5Wl1BfGJJh5sK6Ah/dM
+Tc6XiNl1wSBi1765VVWPiDCScusZoXj+2C3WVB7dtBSPkhlYHMbpasTHqdt421qp
+MKTWnWkZDKpQ3Gg/Ay1qk1HVurGd4sPcuDkB0YDLpHfAwNcyG+FTpbFKugdCZubU
+bjOZ3r5TqK6e1UjAb12QGQKBgCVIfFrgtBsA2bjcYyTl7TtBMzZ0+1Aod3HyIMUZ
+TiD4gQkusws/axmrxOPlnWe0hwCZNhvIAtpXN7Om7ZGEjbUfELKrIwUNnPwMGGGO
+znP9i0x92rqk5B+wemoVi2ybAYtApqA7sRUBoSof6WsaKxWOOJJMbqZwLpnS9iQX
+WWpXAoGBAMegPMaRNKnwrZtXbFuXNrtDo5jcySjevqK0/fSmKZ0jaHaYtpEU9w5s
+hQtW70fa+a18aB6bAx0Mcs89AzW6S50DuW6lJe5YzwHOg79o36DxOPsF5CcoUuOO
+QXTsmgdf/itvilm9n0JiqAM6xu2QmPJUAvTX6JC3o7/3o41+LDuk
+-----END RSA PRIVATE KEY-----
```

Comment:
@DukeHarris please .gitignore your .vagrant folder, we really don't want to index and publish RSA private keys